data/successor 1 introduction.txt

```diff
@@ -218,15 +218,15 @@ mission "Successors: First Contact 2"
 						has "event: successors: tardy for sioeora"
 					goto apologize-tardiness
 			label introduce
-			`	"We are aware of this. Thy introduction is not expected."`
+			`	"We are aware of this. Thy introduction is not unexpected."`
```

Comment:
This was intended as-is; the scions don't expect First Last to introduce themself because the scions already know who they are; and maybe the Successors have some cultural differences over introductions that FL is not aware of.
Perhaps I should make it clearer though?